lib/console/jobs.ex

```diff
@@ -48,7 +48,7 @@ defmodule Console.Jobs do
       end
       recipients = Organizations.get_memberships_by_organization_and_role(alert.organization_id, roles) |> Enum.map(fn (member) -> member.email end)
       details = Enum.map(events, fn (e) -> e.details end)
-      has_hotspot_info = case Enum.find(details, fn d -> d["hotspot"] !== nil end) do
```

Comment:
when `hotspot: {}` in event body, it was still considering this true and then had all empty values in the email